mis_builder/security/ir.model.access.csv

```diff
@@ -20,3 +20,4 @@ access_mis_report_subreport,access_mis_report_subreport,model_mis_report_subrepo
 manage_mis_report_style,access_mis_report_style,model_mis_report_style,account.group_account_manager,1,1,1,1
 access_mis_report_style,access_mis_report_style,model_mis_report_style,base.group_user,1,0,0,0
 access_add_to_dashboard_wizard,access_add_to_dashboard_wizard,model_add_mis_report_instance_dashboard_wizard,base.group_user,1,1,1,0
+access_mis_report_annotation, access_mis_report_annotation,model_mis_report_instance_annotation,base.group_user,1,1,1,1
```

Comment:
That is too broad? Annotation can possibly contain sensitive data so I would not have that readable by all employees by default.

Reply:
To be as flexible as possible, I introduced 2 new groups to edit and read annotations